packages/techdocs-common/src/stages/publish/helpers.ts

```diff
@@ -22,32 +23,16 @@ export type responseHeadersType = {
 /**
  * Some files need special headers to be used correctly by the frontend. This function
  * generates headers in the response to those file requests.
- * @param {string} fileExtension html, css, js etc.
+ * @param {string} fileExtension .html, .css, .js, .png etc.
  */
 export const getHeadersForFileExtension = (
   fileExtension: string,
 ): responseHeadersType => {
-  const headersCommon = {
-    'Content-Type': 'text/plain',
-  };
-  const headersHTML = {
-    ...headersCommon,
-    'Content-Type': 'text/html; charset=UTF-8',
-  };
-
-  const headersCSS = {
-    ...headersCommon,
-    'Content-Type': 'text/css; charset=UTF-8',
-  };
-
-  switch (fileExtension) {
-    case 'html':
-      return headersHTML;
-    case 'css':
-      return headersCSS;
-    default:
-      return headersCommon;
-  }
+  return {
+    'Content-Type': mime.contentType(fileExtension)
```

Comment:
Can this not be `mime.contentType(fileExtension) ?? 'text/plain; charset=utf-8'`?

Reply:
I had tried `??` before, but `mime.contentType(fileExtension)` returns `false` when it does not have any result. And the `leftExpression ?? rightExpression` would only work if `leftExpression` is either `null` or `undefined`.

But, went ahead with using `||` and it is much cleaner. So, thanks!

Reply:
Oh what - that's a pretty strange API. 🤔  Oh well, nicer now.